pkg/handlers/generic/mutation/extraapiservercertsans/inject.go

```diff
@@ -94,6 +94,10 @@ func (h *extraAPIServerCertSANsPatchHandler) Mutate(
 		"variableValue",
 		apiCertSANs,
 	)
+	if len(apiCertSANs) == 0 {
+		log.Info("No APIServerSANs to apply")
```

Comment:
Do we need to this to be always output?
```suggestion
		log.V(4).Info("No APIServerSANs to apply")
```